scripts.js

```diff
@@ -102,17 +104,4 @@ const filterYear = () => {
     oldEnough0 + "<br>" + oldEnough1;
```

Comment:
line 100 - given that `filter` returns an array, it would be more robust and more scalable to map over the results of that array. In this instance you had only two items inside that array, but what if that array grows to say 100 items? Easier to map over them, rather than accessing them individually at their index